CONTRIBUTING.md

```diff
@@ -6,6 +6,14 @@ These are the general guidelines for contributing to WHATWG standards.
 
 See [COMMITTING.md](COMMITTING.md) for further details.
 
+## Getting Started
+
+If this is your first time contributing to the WHATWG standards, check out the global list of [good first issues](https://github.com/search?q=is%3Aopen+label%3A%22good+first+issue%22+user%3Awhatwg&type=Issues) and find one you want to work on. You should comment in the thread to let others know that you want on the issue. This is a great time to ask any questions that you may have.
+
+Contributions to older issues are always appreciated. If an issue hasn't been updated in a while, you should ask if the issue is still relevant before working on it. If someone else was previously working on an issue, and you want to work on it, it's polite to ask that person if you can work on the issue before taking it.
+
+When an issue is inactive or not depends on the type of issue. In general, issues that haven't been updated in 90 days are considered inactive.
```

Comment:
I don't quite understand what this "inactive" concept is meant to convey.

Reply:
Removing this. I just added it as an option, but I don't think it helps much.